README.md

```diff
@@ -10,3 +10,20 @@ Code Coverage
 ```
 npm i @decentralized-identity/ion-sdk --save
 ```
+
+## Additional Setup
+This package depends on the [`@noble/ed25519`](https://github.com/paulmillr/noble-ed25519#usage) and [`@noble/secp256k1`](https://github.com/paulmillr/noble-secp256k1#usage) v2, thus additional steps are needed for some environments:
+
+```ts
+// node.js 18 and earlier, needs globalThis.crypto polyfill
+import { webcrypto } from 'node:crypto';
+// @ts-ignore
+if (!globalThis.crypto) globalThis.crypto = webcrypto;
```

Comment:
if you're going to keep this code inside the library shouldn't this not be in the README?

Reply:
Should the [ed25519 README steps](https://github.com/paulmillr/noble-ed25519#usage) also be included?

Reply:
> if you're going to keep this code inside the library shouldn't this not be in the README?

I've moved it dependencies to tests so it the code would not be loaded under normal library usage.

> Should the ed25519 README steps also be included?

Added. The comment are identical so I thought they were the same...